### ISSUE TITLE
[psautohint] has opinions on what’s bogus

### ISSUE THREAD

**Issue by frankrolf** [adobe-type-tools/afdko#793]
`WARNING: threeemdash: Glyph bounding box looks bogus: 40 235 2230 261.`

Perhaps `psautohint` could check against a list of code points for known-wide glyphs before emitting this message? What’s “bogus” (where is the threshold)?

**Comment by miguelsousa** [comment]
Yes to documenting the threshold. No to adding a hardcoded list.

Any reason why this issue was not created in the [psautohint repo](https://github.com/adobe-type-tools/psautohint)?
